pkg/sdk/integration_test_imports.go

```diff
@@ -0,0 +1,75 @@
+package sdk
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/snowflakedb/gosnowflake"
+)
+
+// All the contents of this file were added to be able to use them outside the sdk package (i.e. integration tests package).
+// It was easier to do it that way, so that we do not include big rename changes in the first moving PR.
+// For each of them we will have to decide what do we do:
+// - do we expose the field/method (e.g. errors or ValidObjectIdentifier)
+// - do we keep the workaround
+// - do we copy the code (e.g. ExecForTests)
+// - do we move the code to other place and use it from both places (e.g. findOne)
+// - something else.
+// This will be handled in subsequent PRs, so that the main difficulty (moving) is already merged.
+
+var (
+	ErrObjectNotExistOrAuthorized = errObjectNotExistOrAuthorized
```

Comment:
i don't have an issue making errors public

Reply:
As discussed I will do it in separate PR.